pkg/dao/repository_configs.go

```diff
@@ -345,15 +347,16 @@ func ModelToApiFields(repoConfig models.RepositoryConfiguration, apiRepo *api.Re
 	apiRepo.Status = repoConfig.Repository.Status
 	apiRepo.GpgKey = repoConfig.GpgKey
 	apiRepo.MetadataVerification = repoConfig.MetadataVerification
+	apiRepo.FailedIntrospectionsCount = repoConfig.Repository.FailedIntrospectionsCount
 
 	if repoConfig.Repository.LastIntrospectionTime != nil {
-		apiRepo.LastIntrospectionTime = repoConfig.Repository.LastIntrospectionTime.String()
+		apiRepo.LastIntrospectionTime = repoConfig.Repository.LastIntrospectionTime.Format(time.RFC3339)
 	}
 	if repoConfig.Repository.LastIntrospectionSuccessTime != nil {
-		apiRepo.LastIntrospectionSuccessTime = repoConfig.Repository.LastIntrospectionSuccessTime.String()
+		apiRepo.LastIntrospectionSuccessTime = repoConfig.Repository.LastIntrospectionSuccessTime.Format(time.RFC3339)
 	}
 	if repoConfig.Repository.LastIntrospectionUpdateTime != nil {
-		apiRepo.LastIntrospectionUpdateTime = repoConfig.Repository.LastIntrospectionUpdateTime.String()
+		apiRepo.LastIntrospectionUpdateTime = repoConfig.Repository.LastIntrospectionUpdateTime.Format(time.RFC3339)
```

Comment:
Small change: changed the format of the timestamps in the API response to be the RFC3339 standard. Still okay to test PR.